uctMaths/competition/reports.py

```diff
@@ -84,13 +84,14 @@ def send_answer_sheets(school, answer_sheet, cc_admin=False):
     recipient_list = [rteacher.email_school]
     if cc_admin:
         recipient_list.append(compadmin.admin_emailaddress())
-    send_email(
+    email = EmailMessage(
         "(Do not reply) UCT Mathematics Competition %s Answer Sheets" % (school.name),
         output_string,
-        'UCT Mathematics Competition <%s>'%(settings.DEFAULT_FROM_EMAIL),#from
-        [{"name": "%s" % (compadmin.get_answer_sheet_name(school)), "value": answer_sheet.getvalue(), "type": "application/pdf"}],
+        'UCT Mathematics Competition <%s>'%(settings.DEFAULT_FROM_EMAIL),
         recipient_list
     )
+    email.attach_file(os.path.join(__file__,"..","..","Declaration", str(school) + "_answer_sheets.pdf"))
+    email.send()
```

Comment:
Is this change necessary? Can we do it the old way please?

Reply:
This doesn't work, background_tasks has request as None, which I think isn't allowed in functions in views.py as the traceback ends on the function call into views.py

Reply:
I don't really understand, there is no `request` anywhere in this function?

Reply:
Whoops, wrong message

Reply:
I mean wrong conversation